en/04_Changelogs/5.0.0.md

```diff
@@ -23,12 +32,68 @@ title: 5.0.0 (unreleased)
+[silverstripe/graphql-devtools](https://packagist.org/packages/silverstripe/graphql-devtools) | CMS5 compatible without commercial support | 
+[silverstripe/html5](https://packagist.org/packages/silverstripe/html5) | Folded back in core | 
+[silverstripe/postgresql](https://packagist.org/packages/silverstripe/postgresql) | CMS5 compatible without commercial support | 
+[silverstripe/recipe-ccl](https://packagist.org/packages/silverstripe/recipe-ccl) | CMS5 compatible without commercial support | 
+[silverstripe/recipe-solr-search](https://packagist.org/packages/silverstripe/recipe-solr-search) | CMS5 compatible without commercial support | 
+[silverstripe/security-extensions](https://packagist.org/packages/silverstripe/security-extensions) | Folded back in core |
+[silverstripe/spellcheck](https://packagist.org/packages/silverstripe/spellcheck) | Dropped | 
+[silverstripe/sqlite3](https://packagist.org/packages/silverstripe/sqlite3) | CMS5 compatible without commercial support | 
+[silverstripe/widgets](https://packagist.org/packages/silverstripe/widgets) | CMS5 compatible without commercial support | 
+[tijsverkoyen/akismet](https://packagist.org/packages/tijsverkoyen/akismet) | Dropped | 
+[tractorcow/classproxy](https://packagist.org/packages/tractorcow/classproxy) | CMS5 compatible without commercial support | 
+[tractorcow/silverstripe-proxy-db](https://packagist.org/packages/tractorcow/silverstripe-proxy-db) | CMS5 compatible without commercial support |
+[undefinedoffset/sortablegridfield](https://packagist.org/packages/undefinedoffset/sortablegridfield) | Dropped | [symbiote/silverstripe-gridfieldextensions](https://packagist.org/packages/symbiote/silverstripe-gridfieldextensions) provides equivalent functionality.
+
+## Fixed dependencies
+
+Silverstripe CMS relies on third party dependencies to implement many features. We have defined several *fixed dependencies* to provide certainty to Silverstripe CMS project owners and minimise the risk of composer conflicts. These dependencies will not be upgraded to new major versions within the Silverstripe CMS 5 release line.
+
+Review [Silverstripe CMS 5 fixed dependencies](/project_governance/fixed_dependencies) for more details.
+
 ## API changes {#api-changes}
 
-This is a major release and as a result there are a number of breaking API changes. For a full list of these see [upgrading your project](/upgrading/upgrading_your_project). Some specific details about a few of them are below.
+This is a major release and contains many breaking API changes. Deprecation warnings have been added to the latest Silverstripe CMS 4 release to advise you of APIs that have been removed in Silverstripe CMS 5.
+
+[Enable deprecation warnings](/upgrading/deprecations) prior to upgrading to Silverstripe CMS 5 to see if your project relies on any APIs that are no longer unavailable.
 
 ### Removed API (by module, alphabetically) {#api-removed}
```

Comment:
Didn't add any API links under this header because there won't be anything to link to since the APIs are gone.

Reply:
That's good, because links under this heading will be automatically generated in a separate PR.